contracts/fast/FastFrontendFacet.sol

```diff
@@ -81,46 +81,45 @@ contract FastFrontendFacet is AFastFacet {
 
   function detailedMember(address member)
       public view returns(MemberDetails memory) {
-    LibFastToken.Data storage tokenStorage = LibFastToken.data();
-    LibFastAccess.Data storage accessStorage = LibFastAccess.data();
     return MemberDetails({
       addr: member,
-      balance: tokenStorage.balances[member],
+      balance: LibFastToken.data().balances[member],
       ethBalance: member.balance,
-      isGovernor: accessStorage.governorSet.contains(member)
+      isGovernor: LibFastAccess.data().governorSet.contains(member)
     });
   }
 
   function detailedGovernor(address governor)
       public view returns(GovernorDetails memory) {
-    LibFastAccess.Data storage accessStorage = LibFastAccess.data();
     return GovernorDetails({
       addr: governor,
       ethBalance: governor.balance,
-      isMember: accessStorage.memberSet.contains(governor)
+      isMember: LibFastAccess.data().memberSet.contains(governor)
     });
   }
 
   function paginateDetailedMembers(uint256 index, uint256 perPage)
       external view returns(MemberDetails[] memory, uint256) {
-    LibFastAccess.Data storage accessStorage = LibFastAccess.data();
     (address[] memory members, uint256 nextCursor) =
-      LibPaginate.addresses(accessStorage.memberSet.values, index, perPage);
+      LibPaginate.addresses(LibFastAccess.data().memberSet.values, index, perPage);
     MemberDetails[] memory values = new MemberDetails[](members.length);
-    for (uint256 i = 0; i < members.length; ++i) {
+    uint256 length = members.length;
+    for (uint256 i = 0; i < length;) {
```

Comment:
`for (uint256 i; i < length;) {`  ? using default `0` for uint256... would this be cheaper still?

Reply:
Well if it's a gas optimisation, @afmfe-iul can have a look.
Just so you know - I'm against some of them.
If we're really after gas optimisations, we shouldn't have the `FastFacetHistory` - which is what consumes half of the gas during transfers - and could be removed if our users were tech-savy ;)

The `unchecked` keyword is **to be avoided**, unless we have the money to hire super-soldiers in solidity.
I only use `unchecked` in very specific circumstances. For example consider the following:

```solidity
require(balance >= amount, "Not enough funds or somethn'");
unchecked {
  balance -= amount;
}
```

So far so good. But as the complexity increases, and as multiple developers work and solve conflicts on the file, you could end up with:

```solidity
require(balance >= amount, "Not enough funds or somethin");
balance -= fee;
unchecked {
  balance -= amount;
}
```

Or even, why not:

```solidity
balance -= fee;
unchecked {
  balance -= amount;
}
```

As much as I like gas optimizations, the compiler isn't smart enough to fence the developer against these problems. So I'd rather have our users spend a bit more gas on the first snippet, or on:

```solidity
balance -= fee;
```